djangocms_versioning/admin.py

```diff
@@ -274,17 +274,41 @@ def get_preview_link(self, obj):
 
     get_preview_link.short_description = _("Preview")
 
+    def _get_field_modifier(self, request, modifier_dict, field):
+        method = modifier_dict[field]
+
+        def get_field_modifier(obj):
+            return method(obj, field)
+
+        get_field_modifier.short_description = field
+
+        return get_field_modifier
+
     def get_list_display(self, request):
         # get configured list_display
-        list_display = self.list_display
+        tuple_display = self.list_display
```

Comment:
I can't see why list_display is not used when the contents is just reset to the original variable on line 303: https://github.com/django-cms/djangocms-versioning/pull/275/files#diff-c3366f60f43af1ae0a294533004f87015a920b766af7133ae5ac52c50ad86b29R304